tensorflow/python/compiler/BUILD

```diff
@@ -12,11 +10,9 @@ py_library(
     name = "compiler",
     srcs = ["__init__.py"],
     srcs_version = "PY3",
-    deps = if_windows(
-        ["//tensorflow/python/compiler/tensorrt:trt_convert_windows"],
```

Comment:
@bixia1 are we good to remove that file ?

Reply:
If the PR works then we don't need it. Right?